src/main/scala/mesosphere/marathon/health/HealthCheckWorkerActor.scala

```diff
@@ -73,6 +73,13 @@ class HealthCheckWorkerActor extends Actor with ActorLogging {
       }
     }
 
+  def getPort(task: MarathonTask, check: HealthCheck): Option[Integer] = {
```

Comment:
I had originally inlined this code in doCheck, but the complexity checker complained about that. I can make it private.